errors/es-mx.po

```diff
@@ -0,0 +1,1474 @@
+#, fuzzy
+msgid "Select Auto-detect proxy settings for this network"
+msgstr "Seleccione Detectar preferencias automáticamente"
+
+#: templates/ERR_AGENT_WPAD+html.body.div.div.ul.li:39-1
+msgid "Select Automatically detect settings"
+msgstr "Selecciona la opción Detección Automática de Proxy"
+
+#: templates/ERR_AGENT_WPAD+html.body.div.div.ul.li:47-1
+msgid "Select Use Automatic proxy configuration"
+msgstr "Selecciona la opción Usar Configuración automática de Proxy"
+
+#: templates/error-details.txt+X509_V_ERR_DEPTH_ZERO_SELF_SIGNED_CERT.descr:77
+msgid "Self signed certificate"
+msgstr "Certificado SSL autofirmado"
+
+#: templates/error-details.txt+X509_V_ERR_SELF_SIGNED_CERT_IN_CHAIN.descr:81
+msgid "Self signed certificate in certificate chain"
+msgstr "Certificado SSL autofirmado en cadena de certificado"
+
+#: templates/error-details.txt+X509_V_ERR_SELF_SIGNED_CERT_IN_CHAIN.detail:81
+msgid "Self-signed SSL Certificate in chain: %ssl_subject"
+msgstr "Certificado SSL autofirmado en cadena: %ssl_subject"
+
+#: templates/error-details.txt+X509_V_ERR_DEPTH_ZERO_SELF_SIGNED_CERT.detail:77
+msgid "Self-signed SSL Certificate: %ssl_subject"
+msgstr "Certificado SSL autofirmado: %ssl_subject"
+
+#: templates/ERR_SOCKET_FAILURE+html.body.div.blockquote.p:24-1
+msgid "Socket Failure"
+msgstr "Fallo en Socket"
+
+#: templates/ERR_INVALID_URL+html.body.div.p:27-1
+msgid "Some aspect of the requested URL is incorrect."
+msgstr "Algún aspecto de la URL solicitado es incorrecto."
+
+#: templates/ERR_CANNOT_FORWARD+html.body.div.p:29-1
+#: templates/ERR_CONFLICT_HOST+html.body.div.p:29-1
+#: templates/ERR_ICAP_FAILURE+html.body.div.p:31-1
+#: templates/ERR_INVALID_REQ+html.body.div.p:27-1
+#: templates/ERR_INVALID_URL+html.body.div.p:29-1
+msgid "Some possible problems are:"
+msgstr "Algunos problemas posibles son:"
+
+#: templates/ERR_CACHE_MGR_ACCESS_DENIED+html.body.div.p:27-1
+msgid ""
+"Sorry, you are not currently allowed to request %U from this cache manager "
+"until you have authenticated yourself."
+msgstr ""
+"Lo siento, no puedes a solicitar %U de este Administrador de Caché hasta "
+"que te hayas autenticado."
+
+#: templates/ERR_CACHE_ACCESS_DENIED+html.body.div.p:27-1
+msgid ""
+"Sorry, you are not currently allowed to request %U from this cache until you "
+"have authenticated yourself."
+msgstr ""
+"Lo siento, no puedes a solicitar %U de este Administrador de Caché hasta "
+"que te hayas autenticado."
+
+#: templates/ERR_ZERO_SIZE_OBJECT+html.body.div.p:27-1
+msgid "Squid did not receive any data for this request."
+msgstr "Squid no recibió información a esta solicitud."
+
+#: templates/ERR_UNSUP_REQ+html.body.div.p:27-1
+msgid ""
+"Squid does not support all request methods for all access protocols. For "
+"example, you can not POST a Gopher request."
+msgstr ""
+"Squid no soporta todos los métodos de solicitud para algunos protocolos de "
+"acceso. Por ejemplo, no es posible hacer una solcitud POST a un servidor   "
```

Comment:
Nit: if we can avoid the unnecessary multiple spaces at the end of this line it saves people bandwidth.
```suggestion
"acceso. Por ejemplo, no es posible hacer una solcitud POST a un servidor "
```

Reply:
Understood.